### PR TITLE
Add test fixture for gRPC 0.15

### PR DESCRIPTION
**Pull request by misterpoe**

### CODE REVIEW

**Comment by misterpoe** [comment]
@matthewloring @ofrobots @cristiancavalli PTAL

File: test/hooks/test-trace-grpc.js [hunk 1]
```diff
@@ -230,6 +231,40 @@ Object.keys(versions).forEach(function(version) {
           stream.end();
         });
       });
+    } else {
+      it('should work for new argument orders (unary)', function(done) {
+        common.runInTransaction(function(endTransaction) {
+          client.testUnary({n: 42}, new grpc.Metadata(), {},
```
Comment (matthewloring) [inline test/hooks/test-trace-grpc.js]: What is the empty object here?

Comment (misterpoe) [inline test/hooks/test-trace-grpc.js]: The signature of the function is `makeUnaryRequest(argument, metadata, options, callback)`, where `metadata` and `options` are optional. So the empty object is `options`.

**Comment by matthewloring** [comment]
LGTM.

**Pull request merged**